add.go

```diff
@@ -564,7 +564,17 @@ func (b *Builder) Add(destination string, extract bool, options AddAndCopyOption
 			}
 		}
 		if itemsCopied == 0 {
-			return errors.Wrapf(syscall.ENOENT, "no items matching glob %q copied (%d filtered out)", localSourceStat.Glob, len(localSourceStat.Globbed))
+			ignoreFile := filepath.Join(contextDir, ".containerignore")
+			if _, err := os.Stat(ignoreFile); err != nil {
+				ignoreFile = filepath.Join(contextDir, ".dockerignore")
+				if _, err := os.Stat(ignoreFile); err != nil {
+					ignoreFile = ""
+				}
+			}
```

Comment:
This guess could well be wrong if `buildah add`'s `--ignorefile` flag is used.

Reply:
There is no way to know at this level whether that happened. I would figure most likely, their would not have been an ignorefile.